build/run-me.sh

```diff
@@ -154,6 +171,14 @@ while [ $# -gt 0 ]; do
     opt_append=inject_mcc_files
     ;;
 
+  --kernel-rot-crt)
+    opt_prev=kernel_rot_crt
+    ;;
+
+  --kernel-rot-key)
```

Comment:
ditto.

Reply:
I'll defer to @jjohnson-arm on the question of what the parameter name should be (he specified it in ARMmbed/mbl-docs#107).